ce-samples-post-authn-ui-react/src/components/Organizations.js

```diff
@@ -0,0 +1,23 @@
+import Debug from "./Debug"
+
+export default function Organizations(props) {
+
+  return (
+   <>
+     {!props.loading && (props.error === '') && props.session.organizations.length > 1 &&
```

Comment:
I'd add optional chaining (e.g. `props.session.organizations?.length`) anytime a property of the API response could be undefined; at one point I got an error thrown from this check while testing

Reply:
Good catch, I fixed it.